packages/interactivity/src/vdom.ts

```diff
@@ -142,7 +142,7 @@ export function toVdom( root: Node ): Array< ComponentChild > {
 				( obj, [ name, ns, value ] ) => {
 					const directiveMatch = directiveParser.exec( name );
 					if ( directiveMatch === null ) {
-						warn( `Invalid directive: ${ name }.` );
+						warn( `Found malformed directive name: ${ name }.` );
```

Comment:
"Invalid directive" could be ambiguous. Since we're validating the directive name here, I opted for a more explicit message.